packages/components/organisms/f-header/src/components/NavLink.vue

```diff
@@ -63,6 +66,12 @@ export default {
             type: String,
             default: 'white'
         }
+    },
+    computed: {
+        // adds extra margin where icon is not passed
+        slotPassed () {
```

Comment:
I'd name this so it references the icon, rather than the slot. So call it `hasIcon` or something along those lines